build.gradle

```diff
@@ -10,9 +10,11 @@ buildscript {
 
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.0'
         classpath 'com.netflix.nebula:nebula-release-plugin:6.1.0'
         classpath 'com.netflix.nebula:nebula-bintray-plugin:3.5.2'
+        classpath('org.jfrog.buildinfo:build-info-extractor-gradle:4.5.4') {
```

Comment:
We'll need to force the plugin to a never version due to this issue:
https://github.com/JFrogDev/build-info/issues/101